requirements.txt

```diff
@@ -1,9 +1,10 @@
 taurus-citrine==0.3.0
 requests==2.22.0
 pyjwt==1.7.1
-arrow==0.14.5
+arrow==0.15.4
```

Comment:
oh snap! will I be able to stop using 
`import warnings
warnings.filterwarnings('ignore')`

Reply:
yah i havent been seeing that log anymore!